### PR TITLE
Added account notifications api

### PR DESCRIPTION
**Pull request by matthewrice345**
Account Notifications will allow users to get global notifications for accounts (Campus wide announcements).

### CODE REVIEW

**Comment by hhoang** [comment]
+1000

File: src/main/java/com/instructure/canvasapi/model/AccountNotification.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    public int[] getRole_ids() {
+        return role_ids;
+    }
+
+    public String[] getIcon() {
+        return icon;
+    }
+
+    public Date getStartDate() {
+        if(start_at == null) {
```
Comment (joshruesch) [inline src/main/java/com/instructure/canvasapi/model/AccountNotification.java]: Doesn't stringToDate handle this automatically?

File: src/main/java/com/instructure/canvasapi/model/AccountNotification.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            return null;
+        }
+        return APIHelpers.stringToDate(start_at);
+    }
+
+    public Date getEndDate() {
+        if(end_at == null) {
+            return null;
+        }
+        return APIHelpers.stringToDate(end_at);
```
Comment (joshruesch) [inline src/main/java/com/instructure/canvasapi/model/AccountNotification.java]: Doesn't stringToDate handle this automatically?

File: src/main/java/com/instructure/canvasapi/model/AccountNotification.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    @Override
+    public Date getComparisonDate() {
+        return getStartDate();
+    }
+
+    @Override
+    public long getId() {
+        return 0;
```
Comment (joshruesch) [inline src/main/java/com/instructure/canvasapi/model/AccountNotification.java]: Shouldn't this return account_notification_id?

**Pull request merged**